tests/test_component.py

```diff
@@ -208,7 +208,7 @@ def test_component_inside_slot(self):
         class SlottedComponent(component.Component):
             template_name = "slotted_template.html"
 
-            def get_context_data(self, name: str | None = None) -> component.Dict[str, component.Any]:
+            def get_context_data(self, name: component.Optional[str] = None) -> component.Dict[str, component.Any]:
```

Comment:
I think we should import Optional from types here instead of reusing it from another file.

Reply:
Thanks for spotting that! Updated in https://github.com/EmilStenstrom/django-components/pull/437/commits/bacd683c44e3e476097850be5c8f3967d25a275e